src/opensearch_sdk_py/rest/extension_rest_handlers.py

```diff
@@ -42,6 +42,4 @@ def named_routes(self) -> list[str]:
         return self._named_routes
 
     def handle(self, route: str, request: ExtensionRestRequest) -> ExtensionRestResponse:
-        handler = self[route]
-        # TODO error response if no handler found (handler is None)
-        return getattr(handler, "handle_request")(request)
+        return getattr(self[route], "handle_request")(request)
```

Comment:
Generally, `getattr` is an antipattern. I opened https://github.com/opensearch-project/opensearch-sdk-py/issues/40.